modules/pam_timestamp/hmac_openssl_wrapper.c

```diff
@@ -49,13 +49,18 @@
 #include <openssl/evp.h>
 #include <openssl/params.h>
 #include <openssl/core_names.h>
+#include <openssl/rand.h>
 
 #include <security/pam_ext.h>
 #include <security/pam_modutil.h>
 
 #include "hmac_openssl_wrapper.h"
 #include "pam_inline.h"
 
+#ifdef HAVE_GETRANDOM
+#include <sys/random.h>
+#endif
```

Comment:
I'm not sure there is a guarantee that `<sys/random.h>` is always provided along with `getrandom`.

Reply:
Will take a look at the docs, but AFAICS at least on Linux it's provided there. For *BSD I've to check.

Reply:
Looking around a bit, I found, that all systems that support `getrandom` have it in `sys/random.h` with the exceptions being OpenBSD, where it's in `unistd.h`. For some systems additional libraries have to be linked; I updated `configure.ac` accordingly.

NB: `unistd.h` is checked in `configure.ac` for its presence, but existing code already assumes it to be available. Thus no update on that front; but `sys/random.h` got a check using `HAVE_SYS_RANDOM_H` proper.